diff --git a/pkg/foo.go b/pkg/foo.go
--- a/pkg/foo.go
+++ b/pkg/foo.go
@@ -1,4 +1,8 @@
 package foo
+// added a
+// added b
+// added c
+// added d
+// added e
+// added f
-// removed a
-// removed b
 func F() {}
diff --git a/pkg/baz.go b/pkg/baz.go
--- a/pkg/baz.go
+++ b/pkg/baz.go
@@ -1,4 +1,8 @@
 package baz
+// added a
+// added b
+// added c
+// added d
+// added e
+// added f
-// removed a
-// removed b
 func Z() {}
